src/app/state/slices/api/gameboards.ts

```diff
@@ -159,13 +159,13 @@ export const unlinkUserFromGameboard = createAsyncThunk<string, {boardId?: strin
                 const hasAssignedGroups = (assignmentsByMe?.filter(a => a.gameboardId === boardId) ?? []).length > 0;
                 if (hasAssignedGroups) {
                     if (reduxState && reduxState.user && reduxState.user.loggedIn && isAdminOrEventManager(reduxState.user)) {
-                        if (!confirm(`Warning: You currently have groups assigned to ${boardTitle}. If you delete this your groups will still be assigned but you won't be able to unassign them or see the ${siteSpecific("gameboard", "quiz")} in your assigned gameboards or 'My gameboards' page.`)) {
+                        if (!confirm(`Warning: You currently have groups assigned to ${boardTitle}. If you delete this your groups will still be assigned but you won't be able to unassign them or see the ${siteSpecific("question deck", "quiz")} in your assigned ${siteSpecific("question decks", "quizzes")} or ${siteSpecific("My question decks", "Manage assignments")} page.`)) {
```

Comment:
The final `${siteSpecific("My question decks", "Manage assignments")}` seems wrong. Those are two different pages?
Should it be "Set assignments" instead of "My question decks"? It is true that it will not be on "My question decks" either, but if for Ada you think the assignments page is the more useful one to refer to (and I don't disagree!) then I think it should be consistent!

```suggestion
                        if (!confirm(`Warning: You currently have groups assigned to ${boardTitle}. If you delete this your groups will still be assigned but you won't be able to unassign them or see the ${siteSpecific("question deck", "quiz")} in your assigned ${siteSpecific("question decks", "quizzes")} or ${siteSpecific("Set assignments", "Manage assignments")} page.`)) {
```

Reply:
Only staff can do this anyhow, so maybe the wording doesn't matter _much_; but just in case it ends up used more widely.

Reply:
Hmm. I had assumed that `your assigned ${siteSpecific("question decks", "quizzes")}` referred to the Set/Manage Assignments page. I went with "Manage assignments" for Ada because  `/quizzes` doesn't seem to be accessible for teachers via any of the menus (unless I'm missing it somewhere), but this does introduce some redundancy for Ada.

Assuming that "your assigned quizzes" and "Manage assignments" are the same thing, we could just remove the last part of the sentence entirely for Ada?